modules/ding_content/ding_content.features.wysiwyg.inc

```diff
@@ -51,20 +54,18 @@ function ding_content_wysiwyg_default_profiles() {
           'media' => 1,
         ),
       ),
-      'toolbar_loc' => 'top',
-      'toolbar_align' => 'left',
-      'path_loc' => 'bottom',
-      'resizing' => 1,
-      'verify_html' => 1,
-      'preformatted' => 0,
-      'convert_fonts_to_spans' => 1,
-      'remove_linebreaks' => 1,
-      'apply_source_formatting' => 0,
-      'paste_auto_cleanup_on_paste' => 0,
-      'block_formats' => 'p,address,pre,h2,h3,h4,h5,h6,div',
+      'toolbarLocation' => 'top',
+      'resize_enabled' => 1,
+      'default_toolbar_grouping' => 0,
+      'simple_source_formatting' => 0,
+      'acf_mode' => 0,
+      'acf_allowed_content' => '',
```

Comment:
This is a byproduct of contrib module upgrade. Actually in our last Sprint we experienced a lot of changes in the default export that we made, and had to manually adjust it in order not to make "unrelated" changes in our commits.
With this said, I think that core team should make exports of everything after next release - in order to keep exports up to date with updated contrib modules, and in order to not have these mixed up commits from various contributors.